packages/botonic-nlu/src/util/file-system.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import {
+  createWriteStream,
+  readFileSync,
+  writeFileSync,
+  mkdirSync,
+  existsSync,
+} from 'fs';
+import { ENCODINGS } from '../constants';
+
+export async function downloadIntoPath({
+  url,
+  downloadPath,
+}: {
+  url: string;
+  downloadPath: string;
+}): Promise<void> {
+  try {
+    const fileWriter = createWriteStream(downloadPath);
+    const downloadedFile = await axios.get(url, { responseType: 'stream' });
+    downloadedFile.data.pipe(fileWriter);
+    return new Promise((resolve, reject) => {
+      fileWriter.on('finish', resolve);
+      fileWriter.on('error', reject);
+    });
+  } catch (e) {
+    console.error(`Error downloading the file.`);
```

Comment:
we should almost never silence an error. It will make debugging very difficult
I suggest converting the console.errors into the message of an new Exception which you rethrow